torch/_higher_order_ops/utils.py

```diff
@@ -99,7 +99,23 @@ def _maybe_reenter_make_fx(fn):
     if _CURRENT_MAKE_FX_TRACER is not None:
         return reenter_make_fx(fn)
     else:
-        return make_fx(fn)
+
```

Comment:
I'm confused, we were previously tracing with real tensors?

Reply:
yeah, when we eagerly run torch.cond with autograd keys set. Then we'll create_fw_bw_graph for it where we trace with real tensor.